charts/cryostat/templates/cryostat_tls_secret.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if (and (not (.Values.authentication.openshift).enabled) (.Values.oauth2Proxy.service.tls.selfSigned.enabled)) }}
+{{- $fullName := include "cryostat.fullname" . }}
+{{- $cert := genSelfSignedCert $fullName nil nil 365 }}
```

Comment:
Just a question: This means the certificate expires after a year right? This means the users have to rotate the certificate themselves?

Any thoughts about depending on `cert-manager` to manage certificates  like on the operator side?

Reply:
Yea, it'll expire after a year. Users have to rotate it themselves or figure out something else automated. I don't really want to get into adding an external dependency to the Helm chart, since part of the reason for the chart to exist is for users who can't use the Operator, ex. because they don't have full admin control of the cluster or whatever else. If the user needs TLS but can't install the Operator, then this gives them at least something to work with as a starting point, and then they can build whatever other automation they need on top to suit their particular deployment environment.

Reply:
Sounds good! Thanks for explaining! I think a quick note in README about this would be helpful :D